### ISSUE TITLE
o-header: improve search button contrast

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#900]
The focus indicator of the ‘close search bar’ button does not meet the required contrast ratio of 3:1 against background colours. This means that users who rely on the use of a keyboard alone to navigate a page, relying on easily distinguishable focus indicators to understand their location and position on the page, may not be able to in this instance as the added border indicating focus only achieves a contrast ratio of 2.4:1 against the background.

![image-20221012-142137](https://user-images.githubusercontent.com/10405691/204034454-81d064aa-db77-43ca-883f-3068e7ccd7dd.png)

https://docs.google.com/document/d/17-d7DAAORFCaUbiHdFPqvrBerUKbMyhG/edit#bookmark=id.99bnk35vsohq

**Comment by akomiqaia** [comment]
Our [global focus PR](https://github.com/Financial-Times/origami/pull/842) should help with this no? contrast with the black border is 11:1. Unless I misunderstood what they meant with this issue. Another thing that could improve contrast is to [remove opacity](https://github.com/Financial-Times/origami/blob/082a66be45e4716e2ccc89620a00ae67cfa8bc40/components/o-header/src/scss/features/_search.scss#L62-L6) on focus for the close button.

**Comment by notlee** [comment]
You're right, when I made this issue I thought it was referring to the icon itself

**Comment by notlee** [comment]
https://github.com/Financial-Times/origami/pull/842
